### PR TITLE
Cleanup section l-p

### PR DESCRIPTION
**Pull request by Zemurin**
Very light reading, no brains required.

### CODE REVIEW

**Comment by Zemurin** [comment]
This concludes the P section, a good place for a pause.

**Review by Idhrendur — APPROVED** [review]

File: EU4toV2/Source/Mappers/LeaderTraits/LeaderTrait.h [hunk 1]
```diff
@@ -1,30 +1,33 @@
...
-		[[nodiscard]] auto getShock() const { return shock; }
-		[[nodiscard]] auto getManeuver() const { return maneuver; }
-		[[nodiscard]] auto getSiege() const { return siege; }
-		[[nodiscard]] auto getOther() const { return other; }
-		
-		[[nodiscard]] bool matches(const int& leaderFire, const int& leaderShock, const int& leaderManeuver, const int& leaderSiege) const;
+class LeaderTrait: commonItems::parser
+{
+  public:
+	explicit LeaderTrait(std::istream& theStream, std::string theName);
```
[the remark this answers is not in the document]
Comment (Zemurin) [reply EU4toV2/Source/Mappers/LeaderTraits/LeaderTrait.h:10]: Actually, I prefer this order with istream leading as it's always a constant, but didn't go so pedantic to reverse the orders of the constructors that had it ordered differently. AFAIK majority of dual+ constructors across the converters have istream leading.

File: EU4toV2/Source/Mappers/PortProvinces/PortProvinces.h [hunk 2]
```diff
@@ -1,28 +1,27 @@
...
+	void registerKeys();
 
-	private:
-		void registerKeys();
-		
-		std::set<int> whitelist;
-		std::set<int> blacklist;
-	};
-}
+	std::set<int> whitelist;
```
[the remark this answers is not in the document]
Comment (Zemurin) [reply EU4toV2/Source/Mappers/PortProvinces/PortProvinces.h:22]: allow/white, deny/black, master/main, slave/wtf, it makes sense for, and only for people in the USA. The rest of the civilized world doesn't have such issues...

[the remark this answers is not in the document]
Comment (Zemurin) [reply EU4toV2/Source/Mappers/PortProvinces/PortProvinces.h:22]: image
VS folks don't care either...

[the remark this answers is not in the document]
Comment (Zemurin) [reply EU4toV2/Source/Mappers/PortProvinces/PortProvinces.h:22]: compared to 
image

**Pull request merged**